src/modules/universal-discovery/universal.discovery.module.js

```diff
@@ -383,8 +433,10 @@ export default class UniversalDiscoveryModule extends Component {
             contentTypesMap,
             multiple,
             searchResultsPerPage,
+            bookmarksPerPage,
             labels,
             restInfo,
+
```

Comment:
not needed empty line

Reply:
I thought that having empty line before `...items.attrs`, and after a big block of other properties would make `...items.attrs` more transparent.
But I will remove it if you want.